src/common/pages/database-page.tsx

```
@@ -216,22 +211,14 @@ export function DatabasePage() {
-            </TableRow>
-          </TableHeader>
-          <TableBody>
-            {companiesData.map((company, index) => (
-              <TableRow key={company.id}>
-                <TableCell>
-                  {(companiesPage - 1) * itemsPerPage + index + 1}
-                </TableCell>
-                <TableCell>{company.name}</TableCell>
-                <TableCell>
+        <div className="grid grid-cols-2 gap-4">
+          {companiesData.map((company, index) => (
```

ESLint

[error] 'index' is defined but never used.

(@typescript-eslint/no-unused-vars)